packages/@aws-cdk/aws-apprunner/lib/observability-configuration.ts

```diff
@@ -0,0 +1,131 @@
+  readonly traceConfiguration?: TracingVendor;
+
+  /**
+    * The name for the Observability Configuration.
+    *
+    * @default - a name generated by CloudFormation
+    */
+  readonly observabilityConfigurationName?: string;
+}
+
+/**
+ * Represents the AppRunner Observability Configuration
+ */
+export interface IObservabilityConfiguration extends cdk.IResource {
+  /**
+   * The ARN of the Observability Configuration.
+   * @attribute
+   */
+  readonly observabilityConfigurationArn: string;
+
+  /**
+   * The Name of the Observability Configuration.
+   * @attribute
+   */
+  readonly observabilityConfigurationName: string;
+}
+
+/**
+ * The App Runner Observability Configuration
+ *
+ * @resource AWS::AppRunner::ObservabilityConfiguration
+ */
+export class ObservabilityConfiguration extends cdk.Resource implements IObservabilityConfiguration {
+  /**
+   * Import an observability configuration by ARN.
+   */
+  public static fromObservabilityConfigurationArn(scope: Construct, id: string, observabilityConfigurationArn: string): IObservabilityConfiguration {
+    class Import extends cdk.Resource implements IObservabilityConfiguration {
+      public readonly observabilityConfigurationArn = observabilityConfigurationArn;
+      public readonly observabilityConfigurationName = observabilityConfigurationArn.split('/')[1];
+    }
+
+    return new Import(scope, id);
+  }
+
+  /**
+   * Import an observability configuration by name.
+   */
+  public static fromObservabilityConfigurationName(scope: Construct, id: string, observabilityConfigurationName: string):
+  IObservabilityConfiguration {
+    return ObservabilityConfiguration.fromObservabilityConfigurationArn(scope, id, Stack.of(scope).formatArn({
+      service: 'apprunner',
+      resource: 'observabilityconfiguration',
+      resourceName: observabilityConfigurationName,
+    }));
+  }
+
+  /**
+   * True if this configuration is the latest revision among all configurations that share the same name.
+   * @attribute
+   */
+  public readonly latest: cdk.IResolvable;
+
+  /**
+    * The ARN of the Observability Configuration.
+    * @attribute
+    */
+  public readonly observabilityConfigurationArn: string;
+
+  /**
+    * The name of the Observability Configuration.
+    * @attribute
+    */
+  public readonly observabilityConfigurationName: string;
+
+  /**
+   * The revision of the Observability Configuration.
+   * @attribute
+   */
+  public readonly observabilityConfigurationRevision: number;
+
+  public constructor(scope: Construct, id: string, props: ObservabilityConfigurationProps = {}) {
+    super(scope, id, {
+      physicalName: props.observabilityConfigurationName,
+    });
+
+    const resource = new CfnObservabilityConfiguration(this, 'Resource', {
+      observabilityConfigurationName: this.physicalName,
+      ...(props.traceConfiguration && props.traceConfiguration != TracingVendor.NONE ?
+        { traceConfiguration: { vendor: props?.traceConfiguration } } : {}),
```

Comment:
None doesn't need to be a thing. Just pass this prop directly. I know it probably seems weird to have only one potential enum value, but it _looks_ based off the Cfn contract that this won't always be the case.